### PR TITLE
Cleanup liabilities accounting

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

**Review by krtk6160 — APPROVED** [review]

File: src/migrations/20210726211641-simplify-accounting.ts [hunk 1]
```diff
@@ -1,20 +1,21 @@
 module.exports = {
   async up(db) {
-    const ftx_txs = await db
-      .getCollection("medici_transactions")
+    const ftx_txs_custor = await db
```
Comment (dolcalmi) [inline src/migrations/20210726211641-simplify-accounting.ts:3]: do you mean cursor? please use camel case :)

File: src/walletFactory.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const WalletFromUsername = async ({
   return WalletFactory({ user, logger })
 }
 
-export const getFunderWallet = async ({ logger }) => {
-  const funder = await User.findOne({ username: yamlConfig.funder })
-  return WalletFactory({ user: funder, logger })
+export const WalletFromRole = async ({ logger, role }) => {
```
Comment (dolcalmi) [inline src/walletFactory.ts:40]: we dont use it as a class so it is a function, please rename to `walletFromRole` or better `getWalletFromRole`

File: src/entrypoint/exporter.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import express from "express"
 import client, { register } from "prom-client"
+import { WalletFromRole } from "src/walletFactory"
```
Comment (dolcalmi) [inline src/entrypoint/exporter.ts:3]: I guess is because new ts build config that the IDE is using `src/` please use relative import

**Review by dolcalmi — CHANGES_REQUESTED** [review]

**Comment by dolcalmi** [comment]
@nicolasburtey please dont forget commit message rule

File: src/entrypoint/exporter.ts (hunk 3 above)
Comment (nicolasburtey) [reply src/entrypoint/exporter.ts:3]: what has been decided for the import (relative/absolute). can we write this down in the repo somewhere?

**Review by dolcalmi — APPROVED** [review]

**Pull request merged**